docs/application-connector/10-03-application-registry-troubleshooting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Application Registry Troubleshooting
+type: Troubleshooting
+---
+
+## Application Registry - No certificate
+
+While trying to access the Application Registry, you may get an error like this:
```

Comment:
While -> When

Reply:
Why would you call the App. Registry?

Reply:
Use 0 conditionals instead of all the "may(s)".

Reply:
I don't like the 'may', but the only alternative that come to my mind are either to split the sentence into two parts divided by the error code block or use 'may', 'can'. 
If I use a conditional, won't it look like you always get an error if you try to access App. Registry? Like it's the default behaviour?

Reply:
Ok, got it.